src/main/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapper.java

```diff
@@ -459,50 +508,67 @@ void validateIfKNNPluginEnabled() {
         }
     }
 
-    Optional<float[]> getFloatsFromContext(ParseContext context, int dimension) throws IOException {
+    // Returns an optional array of byte values where each value in the vector is parsed as a float and validated
+    // if it is a finite number without any decimals and within the byte range of [-128 to 127].
+    @SneakyThrows
```

Comment:
SneakyThrows is a good annotation to be used in tests. We should not use it on src, as this removes what exception the method may be throwing.

Reply:
got it. updated